home-depot-vanilla/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" >
+      <title>JS dropdown</title>
+      <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <h1>JS Drop Down Menu</h1>
+    <nav id="primary_nav_wrap">
+      <ul id="main-menu">
+        <li id="main-menu-li-1" onmouseover="onHover(this.getAttribute('id'));"><a href="#">Menu 1</a>
```

Comment:
Since you are inline adding the onHover function call, you can really just hardcode the id here, instead of saying `this.getAttribute('id')`